examples/dam.jl

```diff
@@ -0,0 +1,163 @@
+#  Copyright 2015, Vincent Leclere, Francois Pacaud and Henri Gerard
+#  This Source Code Form is subject to the terms of the Mozilla Public
+#  License, v. 2.0. If a copy of the MPL was not distributed with this
+#  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#############################################################################
+# Test SDDP with dam example
+# Source: Adrien Cassegrain
+#############################################################################
+
+
+include("../src/SDDP.jl")
```

Comment:
Yes. This will be the goal of a further PR (which will come right after this one)